auth_totp/migrations/10.0.2.0.0/post-migrate.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+import uuid
```

Comment:
Typically you want to import only the namespace(s) you're using (with obvious exceptions if it starts getting out of hand). So in this case, `from uuid import uuid4` would be better.